cvat/apps/engine/task.py

```diff
@@ -779,6 +779,17 @@ def update_status(msg: str) -> None:
 
     av_scan_paths(upload_dir)
 
+    # If something was uploaded to the raw directory, update content_size in the Data model
+    # raw_data_dir and upload_dir are not always the same
+    raw_data_dir = db_data.get_upload_dirname()
+    if os.path.exists(raw_data_dir):
+        size_or_error = get_paths_sizes([raw_data_dir])[raw_data_dir]
+        if not isinstance(size_or_error, int):
+            slogger.glob.warning(size_or_error, exc_info=True)
```

Comment:
Shouldn't we about the creation if we couldn't determine the size? It seems like a sign that something went wrong.

Reply:
It's true that this isn't critical for CVAT to function, but isn't the point of this feature to be able to accurately track users' use of storage space? If we allow tasks to be created without filling in `content_size`, then some storage will be unaccounted for.